api/FetchEvent.json

```diff
@@ -411,6 +411,57 @@
           }
         }
       },
+      "replacesClientId": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent/replacesClientId",
+          "support": {
+            "chrome": {
+              "version_added": null
```

Comment:
This, `chrome_android`, and `webview_android` should be `false`. https://cs.chromium.org/chromium/src/third_party/blink/renderer/modules/service_worker/fetch_event.idl

Reply:
Opera also.

Reply:
Thanks Joe; done. I also updated Samsung Internet to `false`, as that is Chromium-based too